content/learn/introduction.md

```diff
@@ -21,7 +21,7 @@ There are three actively developed versions of Spring Cloud AWS - depending on w
 
 - 2.3.5 - compatible with Spring Boot 2.4.x and 2.5.x - [Reference Docs](https://docs.awspring.io/spring-cloud-aws/docs/2.3.5/reference/html/index.html)
 - 2.4.1 - compatible with Spring Boot 2.6.x and 2.7.x - [Reference Docs](https://docs.awspring.io/spring-cloud-aws/docs/2.4.1/reference/html/index.html)
-- 3.0.0 (not released yet)  - compatible with Spring Boot 2.6.x and 2.7.x, build on the top of AWS SDK v2 - [Reference Docs](https://docs.awspring.io/spring-cloud-aws/docs/3.0.0-SNAPSHOT/reference/html/index.html)
+- 3.0.1 - compatible with Spring Boot 2.6.x and 2.7.x, build on the top of AWS SDK v2 - [Reference Docs](https://docs.awspring.io/spring-cloud-aws/docs/3.0.1/reference/html/index.html)
```

Comment:
Thanks @amrutprabhu for resurrecting the website. The list of versions must be updated more, including Spring Boot compatibility. See https://github.com/awspring/spring-cloud-aws for complete table.